darwin/dataset/download_manager.py

```diff
@@ -316,15 +366,16 @@ def _update_local_path(annotation: AnnotationFile, url, local_path):
 
     # we modify raw json, as internal representation does't store all the data
     with annotation.path.open() as file:
-        raw_annotation = json.load(file)
+        raw_annotation = json.loads(file.read())
```

Comment:
`orjson` is significantly faster, but doesn't expose a `load` method to operate on `TextWrapperIO` functions.

I did plan to write a wrapper for this to expose the same interface, but decided against it for this quick-win, low hanging fruit ticket.